static/app/views/profiling/content.tsx

```diff
@@ -1,7 +1,56 @@
-type Props = {};
+import styled from '@emotion/styled';
+import {Location} from 'history';
 
-function ProfilingContent(_props: Props) {
-  return <h1>Profiling</h1>;
+import * as Layout from 'sentry/components/layouts/thirds';
+import NoProjectMessage from 'sentry/components/noProjectMessage';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
+import PageHeading from 'sentry/components/pageHeading';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {t} from 'sentry/locale';
+import {PageContent} from 'sentry/styles/organization';
+import useOrganization from 'sentry/utils/useOrganization';
+
+import {ProfileScatterChart} from './landing/chart';
+import {ProfileTable} from './landing/table';
+
+interface Props {
+  location: Location;
 }
 
+function ProfilingContent({location}: Props) {
+  const organization = useOrganization();
+  const dateSelection = normalizeDateTimeParams(location.query);
+
+  return (
+    <SentryDocumentTitle title={t('Profiling')} orgSlug={organization.slug}>
+      <PageFiltersContainer>
+        <NoProjectMessage organization={organization}>
+          <StyledPageContent>
+            <Layout.Header>
+              <Layout.HeaderContent>
+                <StyledHeading>{t('Profiling')}</StyledHeading>
+              </Layout.HeaderContent>
+            </Layout.Header>
+            <Layout.Body>
+              <Layout.Main fullWidth>
+                <ProfileScatterChart {...dateSelection} traces={[]} />
```

Comment:
I wonder if instead of spreading these props, we should pass them as dateTimeParams to ProfileScatterChart? I've seen this behaviour in a bunch of other components and I've been passively thinking about this (probably a topic for TSC), but imo it promotes a bad practice of flattening the API and merging types. Let me see if I can explain myself here...

By merging the type here, we lose the notion of the date selection. Our ProfileScatterChart now receives direct props start/end, startPeriod and utc. This makes little sense on it's own and when used in the component itself, you may wonder what these props mean? Because we've merged the type, the notion of date selection is now implicit and the reviewer needs to know this. 

Another con of this (which is minor, but still) is that because most of our type definitions are defined as types, we end up merging the component props with intersection types which end up hurting tsc performance (you didnt actually do this) + a likely negligible runtime performance overhead to unnecessarily spread the object.

Imagine two components:
```tsx
type ScatterChartProps = {
  traces: Profiling.Trace[];
  colorEncoding: string;
} & DateParams

function ScatterChart({start, end, period, utc, traces, colorEncoding}: ScatterChartProps){
    const betweenDates = computeSomeDate(start, end)
    ....
}

<ScatterChart {...dateParams} colorEncoding={"by frame"} traces={traces} />
```
vs
```tsx
interface ScatterChartProps {
  traces: Profiling.Trace[];
  colorEncoding: string;
  dateParams: DateParams
}

function ScatterChart({dateParams, traces, colorEncoding}: ScatterChartProps){
    const betweenDates = computeSomeDate(dateParams.start, dateParams.end)
    ....
}

<ScatterChart dateParams={dateParams} colorEncoding={"by frame"} traces={traces} />
```
Which one do you prefer?

Reply:
@Zylphrex sounds good to me, lets just track these small tasks so we dont forget. Can you maybe open a task in our jira [board](https://getsentry.atlassian.net/jira/software/projects/PROF/boards/152)